packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -367,7 +367,8 @@ Adds a SAML 2.0 application instance
 | --------- | ----------------------------------------------- | -------- | -------- | ------ | ----------------------------------------  |
 | attributeStatements   | Check [here](http://docs.oasis-open.org/security/saml/Post2.0/sstc-saml-tech-overview-2.0-cd-02.html) for details | Array of [Attribute Statements](#attribute-statements-object) | TRUE     | FALSE  |  |
 | configuredAttributeStatements   | SAML attributes that are inherited from app metadata during app creation and are used to configure group attributes | Array of [Group Attribute Statements](#group-attribute-statements-object) | TRUE     | FALSE  |  |
-| destinationOverride   | Overrides the `destination` setting   | String  | FALSE     | FALSE  |                                           |
+| destinationOverride   | Overrides the `destination` setting   | String  | FALSE     | FALSE  |  |
+| honorForceAuthn       | Set to `true` to prompt the user to re-authenticate if the SAML request has the `ForceAuthn` attribute set to `true`. Set this property to `false` to ignore the `ForceAuthn` attribute in the SAML request.       | Boolean   | FALSE    | FALSE  |  |
```

Comment:
food for thought, this is the help.okta blurb which I think is a little more specific and helpful: `Set to Yes to prompt users for their credentials when a SAML request has the ForceAuthn attribute set to true. Users are prompted to enter their credentials, even if they normally sign in through Desktop SSO. If this option is set to No, the attribute is ignored.`

Reply:
Sure, I can use the same Product doc blurb. I'll just leave the `true` / `false` option since this is a boolean parameter.